superset/assets/src/SqlLab/App.jsx

```diff
@@ -20,14 +20,27 @@ setupApp();
 const appContainer = document.getElementById('app');
 const bootstrapData = JSON.parse(appContainer.getAttribute('data-bootstrap'));
 initFeatureFlags(bootstrapData.common.feature_flags);
-const state = getInitialState(bootstrapData);
+const initialState = getInitialState(bootstrapData);
+const sqlLabPersistStateConfig = {
+  paths: ['sqlLab'],
+  config: {
+    slicer: paths => (state) => {
+      const subset = {};
+      paths.forEach((path) => {
+        delete state[path].common; // eslint-disable-line no-param-reassign
```

Comment:
This is just a temporary solution for migrating over existing `localStorage`, and eventually it will be dead code. Do you foresee this happening for any other field currently under `sqlLab` if we decide to change the schema?

Reply:
yes. this line is to fix the screwed localStorage.